src/batch.js

```diff
@@ -1,28 +1,221 @@
 'use strict'
 
+const router = require('lambda-router/src/router')
```

Comment:
This creates a circular dependency. The shared code needs to either be moved into a seperate module, or injected from router into batch